### PR TITLE
translate¿

### PR DESCRIPTION
**Pull request by ddzzkunye**
zh_CN!

### CODE REVIEW

File: filter-nm/lang/zh_CN.ts [hunk 1]
```diff
@@ -36,20 +36,21 @@
         <translation type="unfinished"></translation>
     </message>
     <message>
-        <source>Natural movement filter by Tom Brazier: Cancels higher frequency noise and the natural tendency for our heads to drift even when we think we are sitting still.</source>
-        <translation type="unfinished"></translation>
+        <source>Natural movement filter by Tom Brazier: 过滤高频噪声与我们头部的自然飘移, 即便我们认为我们保持不动.</source>
```
Comment (ddzzkunye) [inline filter-nm/lang/zh_CN.ts:39]: need change

File: filter-nm/lang/zh_CN.ts [hunk 2]
```diff
@@ -37,19 +37,20 @@
     </message>
     <message>
         <source>Natural movement filter by Tom Brazier: Cancels higher frequency noise and the natural tendency for our heads to drift even when we think we are sitting still.</source>
-        <translation type="unfinished"></translation>
+        <translation >Natural movement filter by Tom Brazier: 过滤高频噪声与我们头部的自然飘移, 即便我们认为我们保持不动.</translation>
```
Comment (ddzzkunye) [inline filter-nm/lang/zh_CN.ts:40]: okay

**Review by ddzzkunye — COMMENTED** [review]
okay

File: filter-nm/lang/zh_CN.ts (hunk 1 above)
Comment (GO63-samara) [reply filter-nm/lang/zh_CN.ts]: You translated the string `<source> ... </source>`
But you need to translate the string `<translation> ... </translation>` instead

Comment (ddzzkunye) [reply filter-nm/lang/zh_CN.ts]: It has been fixed.

**Comment by ddzzkunye** [comment]
Why do these files have a 'ts' extension?  
Should I treat it as xml?

**Comment by ddzzkunye** [comment]
Just xml...  
All checks have passed...

**Comment by sthalik** [comment]
> Why do these files have a 'ts' extension?

> Should I treat it as xml?

Use the Qt Linguist from an official Qt installer, or build Qt on your own. https://download.qt.io/archive/qt/5.12/5.12.12/qt-opensource-windows-x86-5.12.12.exe.mirrorlist.

To test it then you'd need to compile opentrack which generates a `zh_CN.qm` file.

Are you sure about removing the degrees per second and mm per second labels?

**Comment by sthalik** [comment]
There's going to be a new release with the Tobii tracker. The translations should definitely go in as well.

**Comment by ddzzkunye** [comment]
> Use the Qt Linguist from an official Qt installer, or build Qt on your own. https://download.qt.io/archive/qt/5.12/5.12.12/qt-opensource-windows-x86-5.12.12.exe.mirrorlist.
> 
> To test it then you'd need to compile opentrack which generates a `zh_CN.qm` file.
> 
> Are you sure about removing the degrees per second and mm per second labels?

I'm sure.

**Pull request merged**